visualization/include/pcl/visualization/impl/point_cloud_geometry_handlers.hpp

```diff
@@ -150,6 +157,9 @@ pcl::visualization::PointCloudGeometryHandlerSurfaceNormal<PointT>::getGeometry
   }
 }
 
+} // namespace visualization
```

Comment:
Maybe have the defines also inside the namespace declarations, or was this causing the compilation errors?

Reply:
Compilation errors for the PCL_INSTANTIATE_* macros.